inst/pages/19_visualization_techniques.qmd

```diff
@@ -349,7 +349,7 @@ feature and sample clusters in a heatmap, along with further functionality.
 # Agglomerate tse by phylum
 tse_phylum <- mergeFeaturesByRank(tse,
                                 rank = "Phylum",
-                                onRankOnly = TRUE)
+                                disable.taxonomy = TRUE)
 
```

Comment:
Use agglomerateByRank

Also this is incorrect example. Remvoe onRankOnly/disable.taxonomy completely